From 3c05d91f252cb109f451a77513fc04b4a9fce728 Mon Sep 17 00:00:00 2001
From: Marcel Metz <mmetz@adrian-broher.net>
Date: Wed, 14 Sep 2016 16:20:31 +0200
Subject: [PATCH] Disable MSVC specific compiler warnings

* Disable _CRT_SECURE_NO_WARNINGS about deprecated stdlib
  functions.
* Disable C4244 'conversion' conversion from 'type1' to 'type2',
  possible loss of data.
* Disable C4305 'identifier' : truncation from 'type1' to 'type2'.
---
 lib/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/lib/CMakeLists.txt b/lib/CMakeLists.txt
index f203b85..16fa575 100644
--- a/lib/CMakeLists.txt
+++ b/lib/CMakeLists.txt
@@ -62,6 +62,8 @@ if(MSVC)
     list(APPEND VORBIS_SOURCES ../win32/vorbis.def)
     list(APPEND VORBISENC_SOURCES ../win32/vorbisenc.def)
     list(APPEND VORBISFILE_SOURCES ../win32/vorbisfile.def)
+    add_definitions(-D_CRT_SECURE_NO_WARNINGS)
+    add_compile_options(/wd4244 /wd4305)
 endif()
 
 include_directories(../include)
-- 
2.7.4

